Makefile

```
@@ -1,7 +1,11 @@
-PROJECT = concourse-helm3
-ID = YOUR_DOCKER_HOST_HERE/${PROJECT}
+PROJECT = concourse-helm-eks-resource
+ID = rcastagnet/${PROJECT}
+#ID = nefelinetworks/${PROJECT}
```

I updated the permissions on the `nefelinetworks/concourse-helm3-resource` repo. Can you update?

```suggestion
ID = nefelinetworks/${PROJECT}
```